docs/user-guide/certificates-configuration-questionnaire.md

```diff
@@ -0,0 +1,118 @@
+- File-based keystore/truststore pair
+- IBM keyring
+
+2. Certificate file format.
+
+You need to acquire or generate certificates in the format accepted by the selected storage type.
+Vice-versa, if you already have digital certificates for your servers, you'd need to select the appropriate storage type that can host them. 
+The digital certificate types that can be used by Zowe are: 
+- PKCS12
+- JKS / JCEKS
+
+3. Certificate signing
+
+Every digital certificate must be signed by a public/private Certificate Authority(CA), or they can be self-signed. Recursively, the CA's signing certificates also must be signed, 
+effectively building a trust chain, rooted by a trusted Root Certificate Authority (who's certificate is self-signed). The Root CA can be a public or a private one.     
+
+When the certificates are signed by a public CA, their genuineness can be easily verified due to the transitive trust policy based on the signing certificates chain and the public availability of that certificates.   
+When the CA is a private to the organization, the clients must explicitly request the CA certificate of the private Root CA and then import it to their trust stores in order to apply the transitive trust policy.   
+
+Choose the type of certificate-signing:
+- Self-signed
+- Private CA
+- Public CA
+
+4. Client or Server validation
+
+Certificates usage can be limited to a server, a client or both. This means that a certificate explicitly created for server usage, can't be used by a client application, and vice-versa a client certificate can't be used by a server component.
+If needed to use the same certificate for both usage types, then it is necessary to either not limit the usage to one type or specify both types in the Extended Usage Attribute. Zowe requires to explicitly state the Client and Server usage in the Extended Usage Atribute of the certificates.
+
+- Only Client validation
+- Only Server validation
+- Both, Client and Server validation
+
+## Certificates configuration questionnaire
+
+Now when you understand the certificates configuration options, answer the following questions
+to choose the best options for certificates type and properties according to your Zowe deployment plan:
+
+1. What is your target environment
+   1. Production - Live production systems open for access from the internet/VPN
+   2. Test (Services, Integrations, Set up, )
+   3. Zowe Development (Enhance Zowe, Develop extensions, Develop services, ...)
+   4. Private system (Learning, Experimenting)
+
+2. Will your installation run on mainframe (z/OS, USS)?
+   1. Yes, all Zowe server components will be hosted on the mainframe.
+   2. No, all Zowe server components will be installed off-platform (z/OS).
+   3. Hybrid approach - some Zowe server components will be installed on-platform and some will be installed off-platform. 
+
+3. What certificates storage type you plan to use? 
+   1. I plan to use z/OS keyring
```

Comment:
Please end both 1 and 2 with a period b/c they're complete sentences. Also do this w/ the answer options to question #4.

Reply:
Done.